docs/content/doc/installation/setup.en-us.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Setup"
+slug: "setup"
+weight: 10
+toc: true
+draft: true
+menu:
+  sidebar:
+    parent: "installation"
+    name: "Setup"
+    weight: 40
+    identifier: "setup"
+---
+
+# Setup
+
+The first time you access the Gitea web interface, the home page will appear. 
+Clicking on "Register" or "Sign In" in the upper right corner will take you to 
+the setup page.
+
+<!-- TODO Describe setup options -->
+<!-- TODO Add screenshots -->
+
+After finishing the setup process, you will be directed to a page where you can 
```

Comment:
Isn't the admin created directly during the install process?

Reply:
Admin creation during installation is optional.

Reply:
Good catch. I haven't gone through the setup process in a while.